Makefile

```
@@ -185,8 +185,15 @@ build: generate fmt vet ## Build manager binary.
 run: manifests generate fmt vet ## Run a controller from your host.
 	go run ./main.go
 
+# Development clusters require linux/amd64 OCI image
+# Set platform to linux/amd64 regardless of host platform.
+# If using podman machine, and host platform is not linux/amd64 run
+# - podman machine ssh sudo rpm-ostree install qemu-user-static && sudo systemctl reboot
+# from: https://github.com/containers/podman/issues/12144#issuecomment-955760527
+# related enhancements that may remove the need to manually install qemu-user-static https://bugzilla.redhat.com/show_bug.cgi?id=2061584
+DOCKER_BUILD_ARGS?=--platform=linux/amd64
```

consistent formatting nit
```suggestion
DOCKER_BUILD_ARGS ?= --platform=linux/amd64
```

Examples:

- https://github.com/openshift/oadp-operator/blob/8bd8c09b5fc5ce1f0737dc91e3d2d639413ef78f/Makefile#L1
- https://github.com/openshift/oadp-operator/blob/8bd8c09b5fc5ce1f0737dc91e3d2d639413ef78f/Makefile#L5
- https://github.com/openshift/oadp-operator/blob/8bd8c09b5fc5ce1f0737dc91e3d2d639413ef78f/Makefile#L115
- https://github.com/openshift/oadp-operator/blob/8bd8c09b5fc5ce1f0737dc91e3d2d639413ef78f/Makefile#L354
- https://github.com/openshift/oadp-operator/blob/8bd8c09b5fc5ce1f0737dc91e3d2d639413ef78f/Makefile#L357